x-pack/test_serverless/functional/test_suites/common/platform_security/navigation/management_nav_cards.ts

```diff
@@ -109,7 +106,7 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       describe('Organization members', function () {
         // Observability will not support custom roles
         // Cannot test cloud link on MKI (will redirect to login)
-        this.tags(['skipSvlOblt', 'skipMKI']);
+        this.tags(['skipSvlOblt', 'skipMKI']); // ToDo: should be able to remove the skipSvlOblt here as custom roles are now supported in OBLT
```

Comment:
This test was failing, probably because this nav card is not being rendered for viewer-role users. However, as there is no way from Kibana to know what cloud privileges a user has, we've defaulted to show this management card for any user in the other solutions (Security, Search). I would recommend keeping this consistent, unless you feel strongly about not showing it.